.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: docker build --no-cache -t image-builder:latest .
       - name: Build system image
         id: build_image
-        run: docker run --rm -v $(pwd)/output:/output --privileged=true image-builder:latest $(echo ${GITHUB_SHA} | cut -c1-7)
+        run: docker run --rm -v $(pwd):/workdir -v $(pwd)/output:/output --privileged=true image-builder:latest $(echo ${GITHUB_SHA} | cut -c1-7)
```

Comment:
does this actually work? I recently noticed that `$(pwd)` doesn't work locally and I had to specify the full path manually.

Reply:
I use the exact same line as here and works for me. By any chance did you set pwd as a variable or are using any other shell? (I'm using bash)